tests/foreman/ui/test_hostunification.py

```diff
@@ -256,6 +257,39 @@ def test_positive_rename_content_host(self):
 
     @run_only_on('sat')
     @tier3
+    def test_positive_rename_content_host_cli(self):
+        """Content Hosts renamed in UI affects entity in the application
+        entirely, so name looks different through CLI too
+
+        :id: a64c7815-269b-45d0-9032-38df550d6cd9
+
+        :steps:
+            1.  Rename a host from 'Content-hosts' page
+            2.  View host under content-hosts
+            3.  View host using command line interface
+
+        :expectedresults: Host changed its name both in UI and CLI
+
+        :BZ: 1417953
+
+        :CaseLevel: System
+        """
+        new_name = gen_string('alphanumeric').lower()
+        with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+            vm.install_katello_ca()
+            vm.register_contenthost(self.org_.label, lce='Library')
+            self.assertTrue(vm.subscribed)
+            with Session(self.browser) as session:
+                session.nav.go_to_select_org(self.org_.name)
+                self.contenthost.update(
+                    vm.hostname,
+                    new_name=new_name,
+                )
+                self.assertIsNotNone(self.contenthost.search(new_name))
+            self.assertIsNotNone(Host.info({'name': new_name}))
```

Comment:
of get the id by the old name at the beginning, and get info at the end by id and check that the name is equal to new_name